pkgs/applications/networking/cluster/terraform-providers/default.nix

```diff
@@ -52,14 +52,43 @@ let
   automated-providers = lib.mapAttrs (_: attrs: mkProvider attrs) list;
 
   # These are the providers that don't fall in line with the default model
-  special-providers = {
+  special-providers = let archived = throw "the provider has been archived by upsteam"; in {
     # Packages that don't fit the default model
     gandi = callPackage ./gandi { };
     libvirt = callPackage ./libvirt { };
     teleport = callPackage ./teleport { };
     vpsadmin = callPackage ./vpsadmin { };
   } // (lib.optionalAttrs (config.allowAliases or false) {
+    arukas = archived; # added 2022/01
+    bitbucket = archived; # added 2022/01
+    chef = archived; # added 2022/01
+    cherryservers = archived; # added 2022/01
+    clc = archived; # added 2022/01
+    cobbler = archived; # added 2022/01
+    cohesity = archived; # added 2022/01
+    genymotion = archived; # added 2022/01
+    hedvig = archived; # added 2022/01
+    ignition = archived; # added 2022/01
+    incapsula = archived; # added 2022/01
+    influxdb = archived; # added 2022/01
+    jdcloud = archived; # added 2022/01
     kubernetes-alpha = throw "This has been merged as beta into the kubernetes provider. See https://www.hashicorp.com/blog/beta-support-for-crds-in-the-terraform-provider-for-kubernetes for details";
+    librato = archived; # added 2022/01
+    logentries = archived; # added 2022/01
+    metalcloud = archived; # added 2022/01
+    mysql = archived; # added 2022/01
+    nixos = archived; # added 2022/01
+    oneandone = archived; # added 2022/01
+    packet = archived; # added 2022/01
+    profitbricks = archived; # added 2022/01
+    pureport = archived; # added 2022/01
+    rancher = archived; # added 2022/01
+    rightscale = archived; # added 2022/01
+    runscope = archived; # added 2022/01
+    softlayer = archived; # added 2022/01
+    telefonicaopencloud = archived; # added 2022/01
+    terraform = archived; # added 2022/01
```

Comment:
https://github.com/hashicorp/terraform-provider-terraform

Reply:
It's archived with no commits since 2018. 

> This repository contains a separated version of the terraform provider that can be used with 0.10.x releases of Terraform Core.
>
> Prior to 0.10, and from 0.11 onwards, this provider is an integrated part of Terraform Core and not packaged as a provider plugin.

So you still wan to keep it?

Reply:
Ok, nevermind.